internal/link/src/index.tsx

```diff
@@ -0,0 +1,60 @@
+import Link from "next/link";
+import React from "react";
+import { cva, type VariantProps } from "class-variance-authority";
+
+const linkStyles = cva(
```

Comment:
@notrab Hey this is super cool! 🚀 

Serge has prepared some style suggestions for our Text Link UI component. Please see the following:

1. https://www.figma.com/design/2e3p7KEFikJjTPXdRyOWfZ/Namekit?node-id=6511-12235&t=GzpgrL9QFr9G5vTl-4
2. https://app.shortcut.com/ps-web3/story/24838/create-reusable-ui-components-for-text-links-and-buttons#activity-25046

Appreciate your advice!